sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_daily_aggregates_v1/init.sql

```diff
@@ -0,0 +1,30 @@
+CREATE OR REPLACE TABLE
```

Comment:
I don't  think you need this file. You can use the schema to init the table

Reply:
hmm... even if I want to specify partitioning settings for the table? 🤔